src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -381,7 +381,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    arg_type=get_enum_type(get_container_access_type_names()),
                    help='Specifies whether data in the container may be accessed publically. By default, container '
```

Comment:
This is way too long for "short help". IMO this help should be moved to `_help.py` and the language starting with "By default..." should be moved to the long summary. 

Nit: "publically" should really be "publicly".

Reply:
Updated,